rocketpy/environment/environment.py

```diff
@@ -276,15 +276,21 @@ def __init__(
         timezone="UTC",
         max_expected_height=80000.0,
     ):
-        """Initialize Environment class, saving launch rail length,
-        launch date, location coordinates and elevation. Note that
-        by default the standard atmosphere is loaded until another
+        """Initialize Environment class, saving parameters of the launch location,
+        such as launch date, coordinates and elevation. This class also computes
+        relevant quantities for the Flight simulation, such as air pressure, density
+        and gravitational acceleration.
```

Comment:
Trying to be more precise with the words here, let me know what you think:

```suggestion
        """Initialize the Environment class, capturing essential parameters of the
        launch site, including the launch date, geographical coordinates, and
        elevation. This class is designed to calculate crucial variables for the
        Flight simulation, such as atmospheric air pressure, density, and
        gravitational acceleration.
```

Reply:
Thanks for the suggestion, applied in [6b84c03](https://github.com/RocketPy-Team/RocketPy/pull/565/commits/6b84c03bdaafd96c140067cfbf0a00fe05556d1e).